src/app/recipe/new/_components/IngredientListItem.tsx

```diff
@@ -0,0 +1,14 @@
+import Link from "next/link";
+
+type Props = {
+  title: string;
+  external?: boolean;
+};
+
+export const IngredientListItem: React.FC<Props> = (props) => {
+  return (
+    <div className="border-b border-lightGray px-4 py-3 last-of-type:border-none">
+      {props.external ? <a href={props.title}>{props.title}</a> : <Link href="/">{props.title}</Link>}
```

Comment:
コメントに漏れあってすみません。
target=_blankつけて、noopener noreferrer属性を付与お願いします！
https://web-plus.jp/noopener-noreferre/

あ、よく見たらhrefにtitle渡っちゃってる...

```suggestion
      {props.external ? <a href={props.title} target="_blank" rel="noopener noreferrer">{props.title}</a> : <Link href="/">{props.title}</Link>}
```